### DISCUSSION TITLE
sf project deploy cancel does not support --target-org parameter

### DISCUSSION THREAD

**Discussion by amarek-cytiva** [forcedotcom/cli#2679] [General]
Hi All,
I was just wondering about the cancel deployment/validation process.
I know that there was a discussion before about the project deploy start, which was also not taking this flag at the beginning because it was using a recently logged-in user.

This has been addressed by adding --target-org parameter which is great.

The problem is now with the cancel command.
According to documentation, it does not allow this parameter which can cause that after having multiple authorizations in the CLI I cannot specify which credentials should be used to cancel the request.

Will it be possible to have the same functionality for the cancel command as well?
It will also make them all consistent in term of parameters they are using, unless there is something else I'm not aware of.

**Comment by WillieRuemmele** [comment]
https://github.com/salesforcecli/plugin-deploy-retrieve/pull/945

**Comment by cristiand391** [comment]
This was included in sf v2.35.6:
https://github.com/forcedotcom/cli/tree/main/releasenotes/#2356-april-3-2024-stable
